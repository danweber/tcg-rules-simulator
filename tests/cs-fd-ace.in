VERSION:2

TITLE: Facedown doesn't overflow
TEXT: Facedown cards don't overflow

FIELD2:Tank,DW1-Big,REST
FIELD1:Mon,DW1-Van,CS2-Rampager,FACEDOWN,CS2-Rampager
GAME:P1 T3 MAIN 10
DUMP:DP

ECHO: -- attack and die, 1 card over flows, the other does not
CMD:ATTACK Mon Tank
STEP:AUTOANSWER:CAPTURE
DUMP:DP


