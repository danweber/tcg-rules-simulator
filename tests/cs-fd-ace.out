FIELD1:Mon,DW1-Vanillamon,CS2-Rampager,FACEDOWN,CS2-Rampager,5000DP
FIELD2:Tank,DW1-Bigmon,REST,12000DP
GAME:P1 T3 MAIN 10
ECHO: -- attack and die, 1 card over flows, the other does not
FIELD1:
FIELD2:Tank,DW1-Bigmon,REST,12000DP
GAME:P1 T3 MAIN 5
